### PR TITLE
Fix broken url

### PR DESCRIPTION
**Pull request by wangjingcun**

### CODE REVIEW

**Comment by slanterns** [comment]
Tbe new link looks... suspicious. And it's not directly viewable either.

File: src/etc/lldb_providers.py [hunk 1]
```diff
@@ -32,7 +32,7 @@
 # You can find more information and examples here:
 #   1. https://lldb.llvm.org/varformats.html
 #   2. https://lldb.llvm.org/use/python-reference.html
-#   3. https://lldb.llvm.org/python_reference/lldb.formatters.cpp.libcxx-pysrc.html
+#   3. https://cgit.geodns-americas.gentoo.org/fork/llvm-project.git/plain/lldb/www/python_reference/lldb.formatters.cpp-pysrc.html?h=release/3.8.x
```
Comment (slanterns) [inline src/etc/lldb_providers.py:35]:
```suggestion
#   3. https://github.com/llvm/llvm-project/blob/llvmorg-8.0.1/lldb/www/python_reference/lldb.formatters.cpp-pysrc.html
```

**Comment by slanterns** [comment]
I suspect this is completely obsolete (since LLVM 9.0). Maybe we can just delete it.

File: src/etc/lldb_providers.py [hunk 2]
```diff
@@ -32,7 +32,7 @@
...
 #   4. https://github.com/llvm-mirror/lldb/tree/master/examples/summaries/cocoa
```
Comment (bjorn3) [inline src/etc/lldb_providers.py:36]: This should probably be changed to point to the official llvm-project repo instead.

File: src/etc/lldb_providers.py (hunk 1 above)
Comment (wangjingcun) [reply src/etc/lldb_providers.py]: Thanks. Modified

**Comment by wangjingcun** [comment]
Anything need I to do?  

I am pleasure to do it.

**Pull request merged**